src/ros_filter.cpp

```diff
@@ -1754,8 +1766,17 @@ namespace RobotLocalization
               tf2::Transform worldBaseLinkTrans;
               tf2::fromMsg(worldBaseLinkTransMsg_.transform, worldBaseLinkTrans);
 
-              tf2::fromMsg(tfBuffer_.lookupTransform(baseLinkFrameId_, odomFrameId_, ros::Time(0)).transform,
-                           odomBaseLinkTrans);
+              if (!RosFilterUtilities::lookupTransformSafe(
+                     tfBuffer_,
+                     baseLinkFrameId_,
+                     odomFrameId_,
+                     ros::Time(filter_.getLastMeasurementTime()),
+                     odomBaseLinkTrans,
+                     true))
+              {
+                ROS_ERROR_STREAM("Unable to retrieve " << odomFrameId_ << "->" << baseLinkFrameId_ << " transform. Skipping iteration...");
```

Comment:
Should this include the last measurement time? If it differs from the log timestamp it could be significant.